ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -921,6 +921,57 @@ def test_tensorflow_max_pool1d(
     )
 
 
+@handle_frontend_test(
+    fn_tree="tensorflow.nn.conv_transpose",
+    x_f_d_df=_x_and_filters(
+        dtypes=helpers.get_dtypes("float", full=False),
+        data_format=st.sampled_from(["NHWC"]),
+        padding=st.sampled_from(["VALID", "SAME"]),
+        stride_min=3,
+        stride_max=7,
+        dilation_max=1,
+        transpose=True,
+    ),
+)
+def test_tensorflow_conv_transpose(
+    *,
+    x_f_d_df,
+    as_variable,
+    num_positional_args,
+    native_array,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    (
+        input_dtype,
+        x,
+        filters,
+        dilations,
+        data_format,
+        stride,
+        pad,
+        output_shape,
+    ) = x_f_d_df
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x,
+        filters=filters,
+        output_shape=output_shape,
+        strides=stride,
+        padding=pad,
+        data_format=data_format,
+        dilations=dilations,
+    )
```

Comment:
Frontend tests have been refactored, the format for the test should be something along the lines of

```
def test_tensorflow_conv_transpose(
    *,
    x_f_d_df,
    frontend,
    test_flags,
    fn_tree,
    on_device,
):
    (
        input_dtype,
        x,
        filters,
        dilations,
        data_format,
        stride,
        pad,
        output_shape,
    ) = x_f_d_df
    helpers.test_frontend_function(
        input_dtypes=input_dtype,
        frontend=frontend,
        test_flags=test_flags,
        fn_tree=fn_tree,
        on_device=on_device,
        atol=1e-1,
        input=x,
        filters=filters,
        output_shape=output_shape,
        strides=stride,
        padding=pad,
        data_format=data_format,
        dilations=dilations,
    )
```